docs/en/index.rst

```diff
@@ -1,10 +1,22 @@
 Event Manager Documentation
 ===========================
 
-Welcome to the Doctrine Event Manager Library documentation.
+The Doctrine Event Manager documentation is a reference guide to everything you need
+to know about the project.
 
-.. toctree::
-   :depth: 2
-   :glob:
+Getting Help
+------------
 
-   *
+If this documentation is not helping to answer questions you have about the
+Doctrine DBAL, don't panic. You can get help from different sources:
+
+-  Gitter chat room `#doctrine/event-manager <https://gitter.im/doctrine/event-manager>`_
+-  On `Stack Overflow <http://stackoverflow.com/questions/tagged/doctrine-event-manager>`_
+-  The `Doctrine Mailing List <http://groups.google.com/group/doctrine-user>`_
+-  Report a bug on `GitHub <https://github.com/doctrine/event-manager/issues>`_.
+
+Getting Started
+---------------
+
+The best way to get started is with the :doc:`Introduction <reference/index#introduction>` section
+in the documentation. Use the sidebar to browse other documentation for the Doctrine PHP DBAL.
```

Comment:
typo: DBAL

Reply:
I don't see the typo?

Reply:
Sorry, it's not DBAL, Its event manager.